sql/parser/builtins.go

```diff
@@ -117,6 +117,18 @@ var builtins = map[string][]builtin{
 		}, typeInt),
 	},
 
+	"char_length": {
+		stringBuiltin1(func(s string) (Datum, error) {
+			return DInt(len(s)), nil
```

Comment:
I think `char_length` should be an alias for `length` which is implemented using `utf8.RuneCountInString`.